Warcraft-GoA-Development-Team/Warcraft-Guardians-of-Azeroth-2
# Definite titular titles

## Changelog:
- Adapted all titular titles by giving an adjective and an article
- Sort this files: wc_other_titles_l_english.yml, wc_holy_order.txt, wc_titular_titles.txt, wc_religious_titles.txt
- Added new english localization
- Added french localization

# How to test:
- Open the game
- Check the titular titles, the holy orders and the religious titles

#  Check if:
- Unused titles from wc_other_titles_l_english.yml are really unused.
- If the new titles names look good. I added suffixes to many of them (for example: Blackrock clan rather than just Blackrock). We have to see if it looks good.
